src/MEDS_transforms/aggregate_code_metadata.py

```diff
@@ -99,6 +102,25 @@ class MapReducePair(NamedTuple):
     reducer: Callable[[pl.Expr | Sequence[pl.Expr] | cs._selector_proxy_], pl.Expr]
 
 
+def quantile_reducer(df: pl.LazyFrame) -> pl.Expr:
+    agg = "values/quantiles"
+    cols = cs.matches(f"{agg}/shard_\\d+")
+    quantile_df = df.select(~cols, pl.concat_list(cols.fill_null([])).alias(agg)).explode(agg).unnest(agg)
+    quantile_df = (
+        quantile_df.with_columns(pl.col("numerical_value").repeat_by("count"))
+        .explode("numerical_value")
+        .drop("count")
+    )
+    quantile_keys = [0.25, 0.5, 0.75]
```

Comment:
We need to find a way to parametrize this. That is the real challenge here, I think, from an interface perspective. These three random quantiles aren't what people will actually want.